doc/api/n-api.md

```diff
@@ -4631,8 +4631,6 @@ that points to its own memory allocated by a native module). Registering
 externally allocated memory will trigger global garbage collections more
 often than it would otherwise.
 
-<!-- it's very convenient to have all the anchors indexed -->
```

Comment:
This is the only document that has it this way, and has been disabled for 2 years